app/javascript/dashboard/store/modules/helpCenterPortals/mutations.js

```diff
@@ -0,0 +1,55 @@
+import Vue from 'vue';
+import { defaultHelpCenterFlags } from './index';
+
+export const mutations = {
+  setUIFlag($state, uiFlags) {
+    $state.uiFlags = {
+      ...$state.uiFlags,
+      ...uiFlags,
+    };
+  },
+
+  addHelpCenterEntry($state, helpCenter) {
+    if (!helpCenter.id) return;
+
+    Vue.set($state.helpCenters.byId, helpCenter.id, {
+      ...helpCenter,
+    });
+  },
+
+  addHelpCenterId($state, helpCenterId) {
+    $state.helpCenters.allIds.push(helpCenterId);
+  },
+
+  updateHelpCenterEntry($state, helpCenter) {
+    const helpCenterId = helpCenter.id;
+    if (!helpCenterId) return;
```

Comment:
Why do we need this check? Sending an update action without an id might not be done anywhere. Even if it was sent, the next line would catch it.